src/assets/css/index.css

```diff
@@ -111,16 +111,16 @@
   }
 
   .scrollbar-table::-webkit-scrollbar {
-    height: 6px;
-    width: 6px;
+    height: 4px;
```

Comment:
Scrollbar might actually be 3px instead: https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6?node-id=15835-25459&m=dev#875251238